qemu/tests/hv_info_check.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    HyperV Enlightenments info query test:
+
+    1) Boot guest.
+    2) Query supported HyperV Enlightenments by host.
+    """
+    cpu_model_flags = params["cpu_model_flags"]
+
+    vm = env.get_vm(params["main_vm"])
+    error_context.context("Query supported HyperV Enlightenments "
+                          "by host", logging.info)
```

Comment:
Sorry I missed this one, `logging` does not work now, please use `test.log` instead.

Reply:
oh, I remember this mentioned in our weekly meeting  : test.log.info() – replace logging.info()
sorry for forget use it this time. Changed it.